changelog.md

```diff
@@ -0,0 +1,5 @@
+Github [handlers] 
+* [github_organization_repo_create_v1] (2020-10-13)
+    * Added success output and error message handling.
```

Comment:
I didn't see the `success` output addition.

Reply:
It wasn't called "success" the output was actually called "output".  output is only populated when the request response code is 200.  I can remove "success" to make it clearer what was added.

Reply:
Removed success to better reflect the changes to the handler.

Reply:
I don't follow this logic. What is the point of only adding the result 'output' when the response code is 200?